service/ingressconfig/v2/key/key.go

```diff
@@ -0,0 +1,20 @@
+package key
+
+import (
+	"github.com/giantswarm/apiextensions/pkg/apis/core/v1alpha1"
+	"github.com/giantswarm/microerror"
+)
+
+func ToCustomObject(v interface{}) (v1alpha1.IngressConfig, error) {
```

Comment:
The key package wasn't a thing here so far but now we have it. We can improve more code later on with this in the resources.